modules/kafka/src/main/java/org/testcontainers/containers/KafkaContainerCluster.java

```diff
@@ -0,0 +1,87 @@
+package org.testcontainers.containers;
+
+import java.util.Collection;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+import static org.testcontainers.containers.KafkaContainer.CONFLUENT_PLATFORM_VERSION;
+
+/**
+ * Provides an easy way to launch a Kafka cluster with multiple brokers.
+ */
+public class KafkaContainerCluster implements AutoCloseable {
+
+    private final int startPort;
+    private final Network network;
+    private final GenericContainer zookeeper;
+    private final Collection<KafkaContainer> brokers;
+
+    public KafkaContainerCluster(int brokersNum, int internalTopicsRf, int startPort) {
+        this(CONFLUENT_PLATFORM_VERSION, brokersNum, internalTopicsRf, startPort);
+    }
+
+    public KafkaContainerCluster(String confluentPlatformVersion, int brokersNum, int internalTopicsRf, int startPort) {
+        if (brokersNum < 0) {
+            throw new IllegalArgumentException("brokersNum '" + brokersNum + "' must be greater than 0");
+        }
+        if (internalTopicsRf < 0 || internalTopicsRf > brokersNum) {
+            throw new IllegalArgumentException("internalTopicsRf '" + internalTopicsRf + "' must be less than brokersNum and greater than 0");
+        }
+
+        this.startPort = startPort;
+
+        this.network = Network.newNetwork();
+
+        this.zookeeper = new GenericContainer("confluentinc/cp-zookeeper:" + confluentPlatformVersion)
+            .withNetwork(network)
+            .withNetworkAliases("zookeeper")
+            .withEnv("ZOOKEEPER_CLIENT_PORT", String.valueOf(KafkaContainer.ZOOKEEPER_PORT));
```

Comment:
I guess there is no need to reuse this constant since we both set & read it from this method